.golangci.yml

```diff
@@ -1,19 +1,36 @@
 linters:
-  enable:
-    - gofmt
-    - govet
-    - errcheck
-    - staticcheck
-    - gosimple
-    - ineffassign
-    - typecheck
-    - unused
-    - misspell
-    - gocyclo
-    - gosec
-    - revive
+  enable-all: true
+  disable:
+    - exportloopref # Deprecated
+    - varnamelen # Not relevant
+    - exhaustruct # Not relevant
+    - ireturn # Not relevant
+    - nlreturn # Not relevant
+    - contextcheck # Too strict
+    - err113 # Too strict
+    - forcetypeassert # Too strict
+    - tagliatelle # Too strict
+    - wrapcheck # Too strict
+    - wsl # Too strict
+    - noctx # Too strict
+    - prealloc # Not maintenant linter
```

Comment:
Very useful one, should be enabled. Not preallocating when you could is a huge performance sink